src/transaction/constants.ts

```diff
@@ -1,3 +1,10 @@
-export const SEND_CALLS_NOT_SUPPORTED_ERROR = 'SEND_CALLS_NOT_SUPPORTED_ERROR';
+export const TRANSACTION_ERROR_CODE = 'WRITE_TRANSACTION_ERROR';
+export const TRANSACTIONS_ERROR_CODE = 'WRITE_TRANSACTIONS_ERROR';
```

Comment:
nit but was wondering if these should be `WRITE_CONTRACT_ERROR` AND `WRITE_CONTRACTS_ERROR` instead of txns

Reply:
Agreed and updated!